src/files/infrastructure/FileJSDataverseRepository.ts

```diff
@@ -172,15 +174,16 @@ export class FileJSDataverseRepository implements FileRepository {
   }
 
   private static getThumbnailById(id: number): Promise<string | undefined> {
-    return fetch(`${this.DATAVERSE_BACKEND_URL}/api/access/datafile/${id}?imageThumb=400`)
-      .then((response) => {
-        if (!response.ok) {
-          throw new Error('Network response was not ok')
-        }
-        return response.blob()
+    return axiosInstance
```

Comment:
I think this was a decision to avoid creating a use case just for this (I wasn't around at the time) and the idea is to keep it that way, I just switch from using a simple fetch to the axios instance I created which automatically sends the authentication token.